plugins/nf-amazon/src/main/nextflow/cloud/aws/config/AwsS3Config.groovy

```diff
@@ -70,6 +70,12 @@ class AwsS3Config implements ConfigScope {
     """)
     final Integer maxConnections
 
+    @ConfigOption
+    @Description("""
+        The maximum size for the heap memory buffer used by concurrent downloads (default:`400 MB`).
+    """)
+    final MemoryUnit maxDownloadBuffer
```

Comment:
I'd call this `maxDownloadHeapBuffer` or `maxDownloadHeapMemory`

Reply:
Considering it's only for the download i've renamed to `maxDownloadHeapMemory` 👉 32b5f105